### ISSUE TITLE
[Feature request] Russian translation

### ISSUE THREAD

**Issue by my-ijet** [olivierkes/manuskript#358]
Please add russian language translation.
(I'm interested in weblate version).

**Comment by gedakc** [comment]
Thank you @my-ijet for the Russian translations updates.

Would you be able to test the current **develop** branch to see if I have correctly added your recent translation work?

**Comment by my-ijet** [comment]
I will check in a few days

**Comment by gedakc** [comment]
I've performed a brief test using **Edit -> Settings -> General (tab)** and by setting **Language** to **Russian**, and then restarting Manuskript.  After that the Manuskript menu entries do appear to change.

I plan to include your translation by creating a **0.7.0** Mansuskript release soon.

**Comment by my-ijet** [comment]
It seems that not all of the lines are translated. Not only in Russian translation.

**Comment by my-ijet** [comment]
In the master branch, however, everything works fine.

**Comment by gedakc** [comment]
Thank you @my-ijet for testing.  Your results are what I expected.

The **master** branch was last updated for the **0.6.0** release and hence does not contain any updates since Nov. 29, 2017.

The **develop** branch contains the latest language translation updates.  The problem is that the `*.ts` translation source files in weblate are old and do not contain strings from recent Manuskript changes.  As you mention this issue exists in other languages too as noted [here](https://github.com/olivierkes/manuskript/issues/230#issuecomment-410414923).

I have updated the `*.ts` translation source files on github.  Unfortunately these updated *.ts files have not automatically been picked up by weblate. Further I do not appear to have permission to push these updates to weblate. :-(

I have emailed @olivierkes requesting an update.

Unfortunately I have not had a response from @olivierkes in a long time.  Consequently in order to get a new release out soon I may have to just accept that not all the strings have been translated and proceed with the release steps.
